ess/src/API/EMBC.Tests.Integration.ESS/Managers/Events/SupportTests.cs

```diff
@@ -474,5 +477,53 @@ public async Task ProcessSelfServeSupports_Supports_SupportsCreated()
             var updatedFile = (await manager.Handle(new EvacuationFilesQuery { FileId = file.Id })).Items.ShouldHaveSingleItem();
             updatedFile.Supports.Count().ShouldBe(supports.Length);
         }
+
+        [Fact]
+        public async Task CreateSelfServeSupportsETransferEmailNoFood()
```

Comment:
The tests should only generate and save the rendered html. You do not want to receive an email every time someone runs the integration tests.